code/data-files/experimentconfig.Any

```diff
@@ -2,8 +2,8 @@
     "settingsVersion" : 1,
     "taskType" : "target",//"target","reaction",
     "expMode" : "real",
-    "expVersion" : "ComplexMotion",
```

Comment:
I don't think we need the changes in this file `experimentconfig.Any` or `userconfig.Any`.